contrib/ci/coverage.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+
+# if invoked outside of CI
+if [ "$CI" != "true" ]; then
+    echo "Not running in CI"
+    exit 1
```

Comment:
I think this should be `exit 0`

Reply:
Not so sure; I think if it's a CI-only script we shouldn't return success.